### PR TITLE
Reorganize scripts

### PR DESCRIPTION
**Pull request by kaustubhkapatral**

### CODE REVIEW

File: deps/env-check.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#/bin/sh
+
+display_usage() {
+    printf "** Please check the exported values:: **\n Daemon : $DAEMON\n Denom : $DENOM\n ChainID : $CHAINID\n DaemonHome : $DAEMON_HOME\n \n Github URL : $GH_URL\n Chain Version : $CHAIN_VERSION\n"
+    exit 1
+}
+
+if [ -z $DAEMON ] || [ -z $DENOM ] || [ -z $CHAINID ] || [ -z $DAEMON_HOME ] || [ -z $GH_URL ] || [ -z $CHAIN_VERSION ]
```
Comment (anilCSE) [inline deps/env-check.sh:8]: Can we check for each env setting separately and show the message clearly. Like, `DAEMON` env setting is missing etc

File: deps/prereq.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#/bin/sh
+command_exists () {
+    type "$1" &> /dev/null ;
+}
+
+cd $HOME
+
+
```
Comment (anilCSE) [inline deps/prereq.sh:8]:
```suggestion
```

File: deps/env-check.sh (hunk 1 above)
Comment (kaustubhkapatral) [reply deps/env-check.sh:8]: This PR is just for moving the scripts into new directories. There will be a new PR up once this gets merged in which all the script refactoring will be included

Comment (anilCSE) [reply deps/env-check.sh:8]: Okay. Merging this for now. Let's address these things clearly in the follow-up PRs

**Review by anilCSE — APPROVED** [review]

**Pull request merged**